talismanrc/scopes.go

```diff
@@ -6,4 +6,5 @@ var knownScopes = map[string][]string{
 	"images":    {"*.jpeg", "*.jpg", "*.png", "*.tiff", "*.bmp"},
 	"bazel":     {"*.bzl"},
 	"terraform": {".terraform.lock.hcl"},
+	"php":       {"composer.lock", "vendor/"},
```

Comment:
Hi Keith,
I worry that vendor folder if checked in could become a source of secret leakage.
How is 'no modifications to vendor' enforced' ?
Why not git ignore it and install on developer machines through some script or out of band sharing ?

Reply:
Hi Suhas,
You are right, the vendor directory is normally .gitignore'd, was following the same pattern as go and node.
Have updated the PR to exclude the vendor dir.
Keith